packages/oidc-middleware/test/integration-test/resources/testRunner.yml

```diff
@@ -18,6 +18,13 @@ scenarios:
     ports:
       applicationPort: 8080
       mockPort: 9090
+      mockHttpsPort: 9999
     command: node
+    env:
+      PORT: 8080
+      ISSUER: https://localhost:9999/oauth2/default
+      CLIENT_ID: OOICU812
+      CLIENT_SECRET: VERY_SECRET
+      NODE_EXTRA_CA_CERTS: ./tck-keystore.pem
```

Comment:
This is node specific configuration, the path to the `tck-keystore.pem` could be arbitrary.

For example the equivalent configuration for the java samples looks like this:
https://github.com/okta/samples-java-spring/blob/master/okta-hosted-login/src/test/resources/testRunner.yml#L27

We could create language framework specific launchers, but that might be overkill 🤔

Reply:
+1 for overkill.

Since we are extracting the cert already from the tck, I would have assumed it would be unnecessary for the configuration object to know about it. Although, this could be a gap of knowledge on my side - 😄

Reply:
It's mostly that any language/framework would have it's own way of configuring the cert (on the mock side of things the cert is automatically configured)